src/Proxy.php

```diff
@@ -14,12 +14,12 @@ interface Proxy
     /**
      * Marker for Proxy class names.
      */
-    public const MARKER = '__CG__';
+    public const string MARKER = '__CG__';
```

Comment:
This interface will be deprecated. I'm not sure it's worth having a breaking change.

Reply:
I don't think that interface is actually extended anywhere, which makes this breaking change very mild (and also, its' really easy to address).